src/views/Proposals.vue

```diff
@@ -129,6 +135,7 @@ export default {
     this.tab =
       this.$route.params.tab || this.space.filters.defaultTab || this.tab;
     this.proposals = await getProposals(this.space);
+    this.spaceUri = await getSpaceUri(this.key);
```

Comment:
This call is a bit heavy (require 3 requests and also loading of ENS pkg), i think it's better to not rely on it and only show settings button for admins.